clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -1959,9 +2013,44 @@ struct CounterCoverageMappingBuilder
                        subtractCounters(ParentCount, TrueCount));
   }
 
+  void RewindDecision(unsigned Since) {
+#ifndef NDEBUG
+    llvm::DenseSet<mcdc::ConditionID> SeenIDs;
+#endif
+    unsigned NConds = 0;
```

Comment:
move the `NConds` into `ifndef NDEBUG` block as it's just a sanity check

Reply:
Thanks. I haven't found it anyways. Will do.